extension/src/paymentHandler/payment-handler.js

```diff
@@ -69,4 +77,9 @@ function _isKlarna (makePaymentRequestObj) {
     && PAYMENT_METHOD_TYPE_KLARNA_METHODS.includes(makePaymentRequestObj.paymentMethod.type)
 }
 
+function _isCancelPayment (paymentObject) {
+  return getAuthorizationTransactionSuccess(paymentObject) &&
+    getCancelAuthorizationTransactionInit(paymentObject)
```

Comment:
If we have previously tried to cancel this same payment, will we have any remaining Init Transactions?

Reply:
If you tried, it should update the status to Pending.